src/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+ul {
+  list-style: none;
```

Comment:
- You do the same in `normalize`
- Also, you set font-family to body, and other font properties to html, why?)